### PR TITLE
persist test cases even if generate grader option is disable

### LINKED ISSUE #379: Some errors for multilang test cases
**Describe the bug / issue**
When a course admin edits the input and output files for test cases in multilang tasks, some errors occur when the task is saved.
e.g. if a file is deleted from the task when it has no test cases, a key error occurs as it cannot find the test cases in task file. Also, we need to verify how the tests files are saved, as when a user made a submission on a task some errors are shown in the submission results.
 
**Expected behavior**
When editing the file system in task, it is necessary to do additional files checks in order to avoid a mishandling of test cases in the task.

**Screenshots**
This is an error when deleting a file of a multiple language task

imagen

**Additional context**
For more information, see closed PR #324.

### PR DESCRIPTION
**Pull request by VirtualDiegox**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
When the option generate grader was checked, the task data wasn't saving the attribute of test_cases, this caused an key error when you try to delete a file related to a test case, now the attributed is saved even if the option generate grader is not checked. 

Fixes #379 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Tested on UNCode deployment in a test course with multilang test task, checking if the option of the generate grader affected the functionality of delete a file in a task or caused an key error.

# Checklist:

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings

### CODE REVIEW

File: inginious/frontend/plugins/grader_generator/pages/grader.py [hunk 1]
```diff
@@ -31,27 +31,30 @@ def on_task_editor_submit(course, taskid, task_data, task_fs):
...
+        return json.dumps({'status': 'error', 'message': error.message})
+
+    # Generate the grader
+    if form.task_data['generate_grader'] and task_data['generate_grader']:
+        form.generate_grader()
+        task_data['grader_test_cases'] = form.task_data['grader_test_cases']
+
+    if not task_data['generate_grader']:
+        #Keep track of test cases saved when generate_grader option is unchecked
+        task_data['grader_test_cases'] = form.task_data['grader_test_cases']
```
Comment (ferestrepoca) [inline inginious/frontend/plugins/grader_generator/pages/grader.py]: Can we refactor this as follows? 

```suggestion
    if form.task_data['generate_grader'] and task_data['generate_grader']:
        form.generate_grader()
    task_data['grader_test_cases'] = form.task_data['grader_test_cases']
```

**Review by ferestrepoca — CHANGES_REQUESTED** [review]
Please revise/test the suggestion and solve i18n conflicts.

File: inginious/frontend/plugins/grader_generator/pages/grader.py (hunk 1 above)
Comment (VirtualDiegox) [reply inginious/frontend/plugins/grader_generator/pages/grader.py]: Yes, works perfect.

**Review by ferestrepoca — APPROVED** [review]
Thanks!

**Pull request merged**